components/splashScreen.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="main-div flex-box">
+    <div class="align-self-center content relative-position">
+      <div class="icon-anim"></div>
+      <div class="flex-box">
+        <SplashIcon class="align-self-center" />
+        <div class="splash-text margin-left-28">TODOS</div>
```

Comment:
we can use i18n for ''TODOS"

Reply:
Should we translate splash screen texts as it is similar to a brand logo?